src/Console/Output/ProcessCommandReporter.php

```diff
@@ -92,4 +93,25 @@ public function reportFileDiffs(array $fileDiffs): void
             $this->consoleStyle->newLine();
         }
     }
+
+    /**
+     * @param Error[] $errors
```

Comment:
You dont use FQCN for doc blocks? This is recommended. We in spryker have a autofixing sniff, but I am sure other packages as well.

Reply:
Why?

I prefer unique class names, that are short and clear without namespace.

Namespaces tend programmers to write names like "App\Product\Manager". Then you have 3 classes named "Manager" in no time. IDE autocomple suffers

Reply:
You need to use sniffing tools to assert it, as we do.
And no, doc blocks must never be linked to code (usage). they must be copy and pastable and still be valid, e.g. in docs and other resources.
And if you use proper tooling about use statement usage you will see the logical error in not using FQCN here. Trust me :)

Reply:
I trust you it works for you. This works for me - I don't copy docs blocks, it's only for static analysis.

And of course I use sniff tools :) 

https://github.com/Symplify/Symplify/blob/8a613ea8e691beb543bc4c5060bfe042bbae64b0/packages/CodingStandard/config/symplify.yml#L6-L11

Reply:
An example of the logical error in your current usage is
https://github.com/spryker/cart-extension/blob/master/src/Spryker/Zed/CartExtension/Dependency/Plugin/CartPreCheckPluginInterface.php
=> `@return \Generated\Shared\Transfer\CartPreCheckResponseTransfer` should not make it necessary in use statements only because of doc block.
Sure, you can maybe add return type hint (here not for BC reasons), but if you e.g. have Type[] collections as array return type it already starts to get really messy.
So again: Highly recommend to rethink that - especially since tooling here can do all the job for you.

Reply:
I still don't get the issue you have.

But I see WTF in duplicated approach - FQN doc but imported namespaced. I'm ok with both, if they're consistent. They'll be moving to typehints (as scalars did) in the future, so future-compat-consistency is a good move from my POV.

Reply:
different topic :) those are usually still needed for additional info or overwrite 
so it is saner to keep it consistent and keep then. again thanks to tooling no effort.